getting-started/2/README.md

```diff
@@ -208,7 +208,7 @@ In our query, we chose to traverse the `follows` edge starting from the selected
 The traversal returns all the nodes connected to the node for `Michael` via the `follows` edge.
 
 3. **Specify the predicates to get back**
-Since `Michael` only follows person, hence, the traversal returns just one node.
+`Michael` only follows one person, hence, the traversal returns just one node.
```

Comment:
I think the following modification sounds better, 
`Since Michael follows only one person, the traversal returns just one node`.